simplify/src/main/java/org/cf/simplify/SimplifyOptionsParser.java

```diff
@@ -108,6 +110,11 @@ private static SimplifyOptions getSimplifyOptions(CommandLine cmd) throws ParseE
             simplifyOpts.setOutFile(new File(outFileName));
         }
 
+        if (Main.errorDexFile != null) {
```

Comment:
As the process restarts it has to load the correct dex file. So this is a trick to do that. If the Main.errorDexFile variable is not null the program has crashed and has to restart. But it doesn't matter as I'm working on just recreating the VM.